recipes/getdns/all/patches/1.6.0-0004-dont-install-symlinked-license.patch

```diff
@@ -0,0 +1,16 @@
+Avoids weird issue on c3i Windows builders:
+    
+  CMake Error at source_subfolder/cmake_install.cmake:104 (file):
+    file INSTALL cannot read symlink
+    "C:/J/w/cci_PR-2903@2/.conan/data/getdns/1.6.0/_/_/build/dd6d0e14f162a68e24ee52fee674d130eac57dea/source_subfolder/COPYING"
+    to duplicate at
+    "C:/J/w/cci_PR-2903@2/.conan/data/getdns/1.6.0/_/_/package/dd6d0e14f162a68e24ee52fee674d130eac57dea/share/doc/getdns/COPYING":
+    File exists.
+  Call Stack (most recent call first):
+
+
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -1080,1 +1080,1 @@
+-install(FILES AUTHORS ChangeLog COPYING LICENSE NEWS README.md DESTINATION ${docdir})
++install(FILES AUTHORS ChangeLog NEWS README.md DESTINATION ${docdir})
```

Comment:
How about keeping the `LICENSE` file? Only the `COPYING` symlink is problematic, right?

(Not sure this makes any difference for the conan recipe; it is probably not the same file as in the license-packaging step, right?)

- https://github.com/conan-io/conan-center-index/pull/2903/files#diff-91193596606c700ab240a997decc0b2243d6eb6adca4ca3f020023a6d91e0c28R125

Reply:
I opted to do the safe way and remove both from the list.
The docdir is removed anyways.